{"errorMessage": "The adapter was unable to infer a handler to use for the event. This is likely related to how the Lambda function was invoked. (Are you testing locally? Make sure the request payload is valid for a supported handler.)", "errorType": "RuntimeError", "requestId": "49a558b0-d252-49db-8bf3-ea9953300a9c", "stackTrace": ["  File \"/var/task/backend/lambda_handler.py\", line 40, in lambda_handler\n    response = handler(event, context)\n", "  File \"/var/task/mangum/adapter.py\", line 62, in __call__\n    handler = self.infer(event, context)\n", "  File \"/var/task/mangum/adapter.py\", line 54, in infer\n    raise RuntimeError(  # pragma: no cover\n"]}